crimsobot/cogs/mystery.py

```diff
@@ -28,6 +28,27 @@ async def tarot(self, ctx: commands.Context) -> None:
         # if no subcommand provided, give a three-card reading
         await self.ppf(ctx)
 
+    @tarot.command(name='one', brief='Get a single card.')  # something better needs to be put here, i do not know much
+    @commands.cooldown(3, 300, commands.BucketType.user)
+    async def one(self, ctx: commands.Context, spread: str = 'one') -> None:
```

Comment:
Is there a reason that `spread` is an argument? Being able to pass a different spread for `>tarot one` (e.g `>tarot one five`) seems odd.

Reply:
Other tarot subcommands do this as well, `spread` is used within the command itself to pass to `tarot.reading()`

Reply:
That's odd behavior. I imagine it's a relic of the command before it was split into subcommands. No doubt @williammck  will see this but pinging anyway. Probably a good idea to think about cleaning this up, though I don't think it's worth opening an issue about.

Reply:
Think I should quickly clean this up and commit it or leave that for later as it's a bit of a larger change?

Reply:
The diff looks ugly enough already, probably better suited for a later change but not sure. Also seems out-of-scope considering the current title for this PR.

Reply:
Got it, will do this later